Specs/getWebGLStub.js

```
@@ -0,0 +1,300 @@
+        stub.renderbufferStorage = noop;
+        stub.sampleCoverage = noop;
+        stub.scissor = noop;
+        stub.shaderSource = noop;
+        stub.stencilFunc = noop;
+        stub.stencilFuncSeparate = noop;
+        stub.stencilMask = noop;
+        stub.stencilMaskSeparate = noop;
+        stub.stencilOp = noop;
+        stub.stencilOpSeparate = noop;
+        stub.texParameterf = noop;
+        stub.texParameteri = noop;
+        stub.texImage2D = noop;
+        stub.texSubImage2D = noop;
+        stub.uniform1f = noop;
+        stub.uniform1fv = noop;
+        stub.uniform1i = noop;
+        stub.uniform1iv = noop;
+        stub.uniform2f = noop;
+        stub.uniform2fv = noop;
+        stub.uniform2i = noop;
+        stub.uniform2iv = noop;
+        stub.uniform3f = noop;
+        stub.uniform3fv = noop;
+        stub.uniform3i = noop;
+        stub.uniform3iv = noop;
+        stub.uniform4f = noop;
+        stub.uniform4fv = noop;
+        stub.uniform4i = noop;
+        stub.uniform4iv = noop;
+        stub.uniformMatrix2fv = noop;
+        stub.uniformMatrix3fv = noop;
+        stub.uniformMatrix4fv = noop;
+        stub.useProgram = noop;
+        stub.validateProgram = noop;
+        stub.vertexAttrib1f = noop;
+        stub.vertexAttrib1fv = noop;
+        stub.vertexAttrib2f = noop;
+        stub.vertexAttrib2fv = noop;
+        stub.vertexAttrib3f = noop;
+        stub.vertexAttrib3fv = noop;
+        stub.vertexAttrib4f = noop;
+        stub.vertexAttrib4fv = noop;
+        stub.vertexAttribPointer = noop;
+        stub.viewport = noop;
+
+        return stub;
+    }
+
+    function noop() {
+    }
+
+    function createStub() {
+        return {};
+    }
+
+    function getStub() {
+        return {};
+    }
+
+    function getStubWarning() {
+        //>>includeStart('debug', pragmas.debug);
+        throw new DeveloperError('A stub for this get/is function is not defined.  Can it use getStub() or does it need a new one?');
```

This is for if we start using `get*` WebGL functions that we are not currently using, we may may need to implement a custom stub for them.  I would rather throw this exception so we know to make the change then try to fly by with `undefined`, `0`, `false`, `{}`, etc.